projects/ti_to_ti/README.md

```diff
@@ -107,4 +113,47 @@ You can set the `INPUT_TYPE` in the `script.sh` to `interleaved-compare` and run
 bash script.sh
 ```
 
-to do the evaluation.
\ No newline at end of file
+to do the evaluation.
+
+## Model Performance
+
+We trained the lm_head of Chameleon-7B model using the full data from [laisonart](huggingface.co/datasets/fantasyfish/laion-art), and acquired a AA-Chameleon-7B model (AA refers to Align-Anything) with image generation capabilities. Then, we used our text-image interleaved input & output dataset (20k) to finetune this model, acquiring a AA-Chameleon-7B+, which is much better in text-image interleaved i/o task. The model is trained for 3 epochs with a batch size of 2 and a learning rate of 5e-05 (similar to meta's setting in [their paper](https://ai.meta.com/blog/generative-ai-text-images-cm3leon)). 
+
+### GPT-4o evaluation
+
+We compared the performance of AA-Chameleon-7B+ with Anole-7b model on a text-image interleaved i/o dataset (with a length of 500). Here's the results:
+
+| win | lose | equal |
+|-------|-------|-------|
+| 240 | 175 | 85 |
```

Comment:
Show the Win Rate, not the number of results

Reply:
fixed in https://github.com/PKU-Alignment/align-anything/pull/41/commits/883bf36ca6c370fca200220a8a1166d320823c7d commit